README.md

```diff
@@ -220,3 +220,7 @@ To build `MapboxDirectionsCLI` using SPM:
 
 1. `swift build`
 1. `swift run MapboxDirectionsCLI -h` to see usage.
+
+## Pricing
+
+API calls made to the Directions API are individually billed by request. Review the [pricing information](https://docs.mapbox.com/api/navigation/directions/#directions-api-pricing) and the [pricing page](https://www.mapbox.com/pricing/#directions) for current rates.
```

Comment:
By the way, it looks like this PR is based on the main branch instead of the release-v2.0 branch. We’ll need to merge main to release-v2.0 sometime soon, but the diff will be cleaner if you rebase onto release-v2.0 in the meantime. Thanks!

Reply:
This change is still in here; it has landed in main but not in release-v2.0 yet because we haven’t merged from main to release-v2.0 in a while. That’s something we can do independently of this PR, even without a separate PR.

Reply:
I merged main to release-v2.0, so another rebase should simplify this PR further.